tools/clang/test/HLSLFileCheck/d3dreflect/bitfield_spill-over.hlsl

```diff
@@ -0,0 +1,141 @@
+// CHECK-NEXT:                   Rows: 1
+// CHECK-NEXT:                   Columns: 1
+// CHECK-NEXT:                   Members: 3
+// CHECK-NEXT:                   Offset: 4
+// CHECK-NEXT:                 {
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 8
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 0
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 4
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 8
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 5
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 12
+// CHECK-NEXT:                 }
+// CHECK-NEXT:               ID3D12ShaderReflectionType:
+// CHECK-NEXT:                 D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                   Class: D3D_SVC_SCALAR
+// CHECK-NEXT:                   Type: D3D_SVT_INT
+// CHECK-NEXT:                   Elements: 0
+// CHECK-NEXT:                   Rows: 1
+// CHECK-NEXT:                   Columns: 1
+// CHECK-NEXT:                   Members: 1
+// CHECK-NEXT:                   Offset: 8
+// CHECK-NEXT:                 {
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 6
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 0
+// CHECK-NEXT:                 }
+// CHECK-NEXT:             }
+// CHECK-NEXT: CBuffer: B
+
+struct BF {
+   int i0 : 8;
+   int i1 : 8;
+   int i2 : 8;
+   int i3 : 8;
+   int i4 : 8; // spill over, start of second primitive
+   int : 4; // what about blank field?  do we support this?
```

Comment:
Great, now we should drop the questions in the comment from my example, since they are answered ;)

Reply:
:). I'll remove it.